onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java

```diff
@@ -65,9 +66,9 @@ protected void onCreate(Bundle savedInstanceState) {
             ObaRegion currentRegion = Application.get().getCurrentRegion();
 
             if (showValidateRegionDialog(currentRegion)) {
+                mRegionId = currentRegion.getId();
```

Comment:
Agreed, this would also be a cleaner separation between the current hosting Activity and the Fragment.